test/e2e/autoscaling/horizontal_pod_autoscaling.go

```diff
@@ -106,18 +110,50 @@ var _ = SIGDescribe("[Feature:HPA] Horizontal pod autoscaling (scale resource: C
 	})
 })
 
+var _ = SIGDescribe("[Feature:HPA] Horizontal pod autoscaling (scale resource: custom metrics)", func() {
+	f := framework.NewDefaultFramework("horizontal-pod-autoscaling")
+	f.NamespacePodSecurityEnforceLevel = api.LevelBaseline
+
+	// Set up FeatureGate for enabling HPAScaleToZero, which is hidden by alpha feature gate.
+	fg := featuregate.NewFeatureGate()
+	fg.Add(map[featuregate.Feature]featuregate.FeatureSpec{"HPAScaleToZero": {
+		Default:       false,
+		LockToDefault: false,
+		PreRelease:    featuregate.Alpha,
+	}})
+	e2eskipper.InitFeatureGates(fg, nil)
+
+	ginkgo.Describe("[Serial] [Slow] Scale down to zero", func() {
+		ginkgo.It("should scale down from 3 pods to zero with custom metrics", func() {
```

Comment:
nit: s/zero/0
We use digits in every other test

Reply:
Done.